doc/admin-api.md

```diff
@@ -688,6 +689,31 @@ Config Example:
 }
 ```
 
+## Global Rule
+
+*API*：/apisix/admin/global_rules/{id}
+
+*Description*: Set plugins which run globally. Those plugins will be run before any Route/Service level plugins.
```

Comment:
and Consumer?

Reply:
We don't execute Consumer plugin. The plugin is used to detect which consumer is matched.